docs/serverless/cloud-native-security/cspm-get-started.mdx

```diff
@@ -41,14 +41,22 @@ You can set up CSPM for AWS either by enrolling a single cloud account, or by en
 1. Click **Add Cloud Security Posture Management (CSPM)**.
 1. Select **AWS**, then either **AWS Organization** to onboard multiple accounts, or **Single Account** to onboard an individual account.
 1. Give your integration a name that matches the purpose or team of the AWS account/organization you want to monitor, for example, `dev-aws-account`.
+1. (Optional) click **Advanced settings** to deploy the integration using agentless technology (beta). 
```

Comment:
Fixes typo, and uses a formal [beta badge](https://docs.elastic.dev/docs/reference/doc-badge):
```suggestion
1. <DocBadge template="beta" /> (Optional) Click **Advanced settings** to deploy the integration using agentless technology. 
```